src/Model/ApiMapper.php

```diff
@@ -78,7 +78,8 @@ public function transformResults(array $results, $verbose)
                     } else {
                         $tz = new DateTimeZone('UTC');
                     }
-                    $entry[$key] = (new DateTime('@' . $row[$value]))->setTimezone($tz)->format('c');
+                    $time = is_numeric($row[$value]) ? '@' . $row[$value] : $row[$value];
+                    $entry[$key] = (new DateTime($time))->setTimezone($tz)->format('c');
```

Comment:
Can we please change this to a `DateTimeImmutable` to avoid problems (or to realize them)